torchbox/wagtail-torchbox
# Add course pages FE

:warning: Based on https://github.com/torchbox/wagtail-torchbox/pull/562

### Description of Changes Made

This PR builds on Kev's BE work, adding FE for the course landing/detail pages. 

No tickets for this feature.

Designs [here](https://www.figma.com/file/H5pQECphbbYsvSWtag3OrCWC/Torchbox?type=design&node-id=3309-9169&mode=design&t=qkiQEHMhodPskH8Z-0) - note we've deviated in some places in order to re-use existing styles, e.g 'What you’ll learn' section.

### How to Test

See https://github.com/torchbox/wagtail-torchbox/pull/562 for instructions on which pages to add and further detail. To review, the reviewer will need to manually create and populate these pages in Wagtail - we're not bothering with PL data given its existing state.

In one or two places I've had to override existing CSS specificity, or pulled in code from the Careers site - it wasn't deemed worth the effort to bring these up to scratch for this codebase given the imminent redesign project.

### Screenshots

![localhost_3001_course-landing-page_ (1)](https://github.com/torchbox/wagtail-torchbox/assets/2553896/c3c5075b-99c0-44c6-97b0-26da2a2e9125)
![localhost_3001_course-landing-page_](https://github.com/torchbox/wagtail-torchbox/assets/2553896/aeac9841-495e-4723-b7da-c4085570fec0)
![localhost_3001_course-landing-page_course-1-django-developer-fundamentals_ (1)](https://github.com/torchbox/wagtail-torchbox/assets/2553896/c486c4f6-7688-40d7-94ff-dce3e4ef054e)
![localhost_3001_course-landing-page_course-1-django-developer-fundamentals_](https://github.com/torchbox/wagtail-torchbox/assets/2553896/89460b36-223b-449b-a780-4693a6c9948d)
### MR Checklist
